### DISCUSSION TITLE
Sign up with Google

### DISCUSSION THREAD

**Discussion by badcom** [pglombardo/PasswordPusher#1488] [Ideas]
How about an option to sign up / in with Google?

**Comment by pglombardo** [comment]
Hi @badcom - I agree that it's needed.   I'll get to it eventually - not sure when just yet though.  Also filed is #410 & #658 which are along the same lines.  Thanks for pointing this out!

**Comment by pglombardo** [comment]
Hi @badcom  - Just an update that Google & Microsoft SSO has been added to pwpush.com.  This will make it to the open source version hopefully soon.
